app/models/school.rb

```diff
@@ -11,15 +7,7 @@ class School < ActiveRecord::Base
 
   def full_name(delim = ', ')
     n = name
-    if !campus.nil?
-      if campus.schools.size > 1
-        n += "#{delim}#{campus.name}"
-      end
-
-      if !campus.borough.nil?
-        n += "#{delim}#{campus.borough.name}"
-      end
-    end
+    n += "#{delim}#{borough}" if !borough.nil?
```

Comment:
Are campuses being removed from the model?

Reply:
@daryanypl we're leaving campus data alone just in case we need it in the future.  We're not importing campus info for new schools, though.